docs/src/wallet-guide/apps.md

```diff
@@ -7,6 +7,30 @@ experience for most people who are new or experienced with using crypto wallets.
 currently the easiest and fastest way to get set up with a new wallet on Solana.
 The app is free and getting your wallet set up only takes a few minutes.
 
+### Trust Wallet Security
+
+Tokens held in Trust Wallet are only as secure as the device on which the app is
+installed.  Anyone who is able to unlock your phone or tablet may be able to
+use the Trust Wallet app and transfer your tokens.  It is *highly* recommended
+that you always use a strong passcode to lock your device and use a different
+passcode to open the app itself.  Enable Trust Wallet passcode by opening
+the app and going to Settings -> Security -> Passcode.
+
+Additionally, anyone who has access to your seed phrase will be able to recreate
+your Trust Wallet keys on a different device, and sign transactions from that
+device, rather than on your own phone or tablet.
+The seed phrase is displayed when a
+new wallet is created, and it can also be viewed at any later time in the app by
+going to Setting -> Wallets, and under the Options menu for a particular wallet,
+tap "Show Recovery Phrase".  If you use an insecure method of electronic backup,
+such as emailing yourself your seed phrase, if the email is intercepted or if
+anyone else has access to the same email account, they would be able to create
+your Trust Wallet keys on another device.
+
+For these reasons, we do not recommend storing large quantities of SOL using
```

Comment:
I don't know if this paragraph is necessary.  The point of the "Security" sections under each wallet is to allow a newcomer to decide which wallet best fits their needs.  If the text above is clear, this part should be obvious.